docs/guides/multicloud_cluster_with_baremetal_servers.md

```diff
@@ -0,0 +1,82 @@
+---
+page_title: "Using Elastic Metal servers in a Kubernetes cluster"
+description: |-
+How to add and/or run Scaleway baremetal servers as nodes on a multicloud Kubernetes cluster
+---
+
+# How to use Elastic Metal servers with a Kosmos multicloud cluster
+
+In this guide you will learn how to deploy baremetal nodes on your Kubernetes cluster instead of instances. To be able
+to do this, you need to have a Kosmos multicloud cluster instead of a Kapsule cluster, and add your nodes as the
+"external" node type.
+
+## Prerequisites
+
+* An SSH key, required to connect to your Elastic Metal server.
+
+## Setup
+
+```hcl
+resource "scaleway_k8s_cluster" "multicloud" {
+  name    = "multicloud-cluster"
+  type    = "multicloud"
+  version = "1.26.0"
+  cni     = "kilo"
+  region  = "fr-par"
+  delete_additional_resources = false
+}
+
+resource "scaleway_k8s_pool" "pool" {
+  cluster_id  = scaleway_k8s_cluster.multicloud.id
+  name        = "multicloud-pool"
+  node_type   = "external"
+  size        = 1
+  region      = "fr-par"
+}
+
+resource "scaleway_baremetal_server" "server" {
+  offer       = "EM-B112X-SSD"
+  os          = "03b7f4ba-a6a1-4305-984e-b54fafbf1681" # Ubuntu_focal
+  ssh_key_ids = [scaleway_iam_ssh_key.key.id]
+  zone        = "fr-par-2"
+}
+
+resource "scaleway_iam_ssh_key" "key" {
+  name = "ssh-key"
+  public_key = file("~/.ssh/id_ed25519.pub")
+}
+```
+
+NB: `kilo` is the only CNI compatible with multicloud clusters
+
+NB: if you want to link any already existing resource, you can import it to the Terraform state by running :
+
+```bash
+terraform import <scaleway_resource_type> <locality>/<id>
+```
+
+## Configure the Elastic Metal server
+
+1. Get your server's public IP and SSH to the server :
+
+    ```bash
+    ssh <user>@<baremetal_server_ip>
+    ```
+
+2. Download the multicloud-init script :
+
+    ```bash
+    wget https://scwcontainermulticloud.s3.fr-par.scw.cloud/multicloud-init.sh && chmod +x multicloud-init.sh`
```

Comment:
Can we use the [remote-exec provisioner](https://developer.hashicorp.com/terraform/language/resources/provisioners/remote-exec) to do this?

Reply:
It is indeed possible, I've added an alternate method of configuration. Thanks for the hint ;)